### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by krzysztof-gzocha**
Example of usage added fixing issue #17 
@XanderEmu would it be a nice example?

### CODE REVIEW

**Comment by XanderEmu** [comment]
Yes it would. It's also very colorful - I love it :)
One think that bothers me is construction of Imposer class. Does a datasource abstraction end on imposer's level? Because that code looks a lot like there's DQL in there (it's not much of a bad thing but it should be made clear).

File: README.md [hunk 1]
```diff
@@ -27,6 +27,89 @@ Integration with Symfony is done in [SearcherBundle](https://github.com/krzyszto
...
+
+    public function supportsModel(
+        FilterModelInterface $filterModel
+    ) {
+        // No need to check isImposed(). Searcher will check it
+        return $filterModel instanceof AgeRangeFilterModel;
+    }
+
+    /**
+    * You can skip this method if you will extends from QueryBuilderFilterImposer.
```
Comment (XanderEmu) [inline README.md]: extends -> extend

**Comment by krzysztof-gzocha** [comment]
`FilterImposer` will know how to impose conditions on specific _QueryBuilder_, so they know if it's Doctrine, Propel, Elastica or whatever and they will integrate with them. They will fetch specific QueryBuilder directly from `SearchingContext` and thats it. Rest of searcher library doesn't know what database is used.

File: README.md (hunk 1 above)
Comment (krzysztof-gzocha) [inline README.md]: :+1:

**Comment by XanderEmu** [comment]
That's fine but please make it clear in the example.
I personally like to name implementations in a way that describes exactly what they are. In this case I could call the imposer e.g. AgeRangeFilterDoctrineImposer.

**Comment by krzysztof-gzocha** [comment]
Thats also a good point. I will fix this maybe even today.

**Pull request merged**